configure.ac

```diff
@@ -180,7 +189,7 @@ dnl $4 - pkg-config module to look for
 dnl $5 - fallback test for the feature
 m4_define([GD_LIB_PKG_CHECK], [dnl
   GD_LIB_CHECK([$1], [$2], [$3], [dnl
-    PKG_CHECK_MODULES([$1], [$4], [gd_found_lib=yes], [$5])
+    PKG_CHECK_MODULES([$1], [$4], [gd_found_lib=yes gd_require_pkg_name=["$4"]], [$5])
```

Comment:
does this need the `[]` around `"$4"` ?  i don't think it does.

Reply:
done